pkgs/development/python-modules/quantile-forest/default.nix

```
@@ -0,0 +1,63 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, python
+, pythonOlder
+
+, setuptools
+, wheel
+, cython_3
+
+, numpy
+, scipy
+, scikit-learn
+}:
+
+buildPythonPackage rec {
+  pname = "quantile-forest";
+  version = "1.3.4";
+  pyproject = true;
+
+  disabled = pythonOlder "3.8";
+
+  src = fetchFromGitHub {
+    owner = "zillow";
+    repo = "quantile-forest";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-hzLJq0y+qjc48PfHW3i73x9safGOy0V1HEQ5WR8IXpI=";
+  };
+
+  build-system = [
+    setuptools
+    cython_3
```

```suggestion
    cython
```
`cython` should refer to cython 3 now.